ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-hosted-pipeline.yml

```diff
@@ -1201,7 +1217,9 @@ spec:
     - name: upload-pipeline-logs
       taskRef:
         name: upload-pipeline-logs
-      when: *certProjectExists
+      when:
+        - *certProjectExists
+        - *bundleAdded
```

Comment:
I think you're putting a `bundleAdded` condition in places where it doesn't belong. Why do we need to skip `upload-pipeline-logs` if a bundle is absent?